src/bind.js

```diff
@@ -18,8 +18,17 @@
  *
  * @return {Function}
  */
-function bind(callback) {
-  // write code here
+function bind(callback, ...numbers) {
+  const numbersArr = [...numbers];
+
+  return (...args) => {
```

Comment:
good, but you can write simply
```suggestion
  return (... args) => callback(... numbers, ... args);
```

Reply:
wow, it's great idea